cart/domain/cart/cartServicePorts.go

```diff
@@ -41,6 +41,7 @@ type (
 	ModifyBehaviour interface {
 		DeleteItem(ctx context.Context, cart *Cart, itemID string, deliveryCode string) (*Cart, DeferEvents, error)
 		UpdateItem(ctx context.Context, cart *Cart, itemID string, deliveryCode string, itemUpdateCommand ItemUpdateCommand) (*Cart, DeferEvents, error)
```

Comment:
Maybe we should think of adding all relevant properties to the ItemUpdateCommand:
- itemId
- deliveryCode

Then the UpdateItems can get a slice of UpdateCommands